charts/karpenter/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v2
 name: karpenter
 description: A Helm chart for https://github.com/aws/karpenter/.
 type: application
-version: 0.5.2
+version: 0.5.3
```

Comment:
can you revert this back to 0.5.2? We'll increment the version during our release process.